Task 3/Longest Common Subsequence/Longest Common Subsequence.cpp

```diff
@@ -0,0 +1,46 @@
+#include<bits/stdc++.h> 
```

Comment:
Can we have a multiline comment at the beginning of this file having a short explanation of the problem with sample input/output cases?

Reply:
@nimishbongale Explanation added